Mathlib/Analysis/Normed/Field/Basic.lean

```diff
@@ -447,6 +447,21 @@ instance MulOpposite.instSeminormedRing : SeminormedRing αᵐᵒᵖ where
   __ := instRing
   __ := instNonUnitalSeminormedRing
 
+lemma norm_sub_mul (ha : ‖a‖ ≤ 1) : ‖c - a * b‖ ≤ ‖c - a‖ + ‖1 - b‖ :=
```

Comment:
`norm_sub_mul_le`? Also, please add an explanation similar to the one in the top comment to the docstring.